CHANGELOG-3.2.md

```diff
@@ -76,6 +76,8 @@ See [code changes](https://github.com/coreos/etcd/compare/v3.2.18...v3.2.19) and
 - Fix [`etcd_debugging_server_lease_expired_total`](https://github.com/coreos/etcd/pull/9557) Prometheus metric.
 - Fix [race conditions in v2 server stat collecting](https://github.com/coreos/etcd/pull/9562).
 - Add [`etcd_server_is_leader`](https://github.com/coreos/etcd/pull/9587) Prometheus metric.
+- Add [`etcd_server_version`](https://github.com/coreos/etcd/pull/8960) Prometheus metric.
```

Comment:
Move all the way to the top?

The backport will be released in the latest patch release :)

> v3.2.23 (TBD 2018-06)